temporalcli/commands.taskqueue.go

```diff
@@ -99,3 +99,44 @@ func (c *TemporalTaskQueueDescribeCommand) run(cctx *CommandContext, args []stri
 	}
 	return cctx.Printer.PrintStructured(items, printer.StructuredOptions{Table: &printer.TableOptions{}})
 }
+
+func (c *TemporalTaskQueueListPartitionCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	request := &workflowservice.ListTaskQueuePartitionsRequest{
+		Namespace: c.Parent.Namespace,
+		TaskQueue: &taskqueue.TaskQueue{
+			Name: c.TaskQueue,
+			Kind: enums.TASK_QUEUE_KIND_NORMAL,
+		},
+	}
+
+	resp, err := cl.WorkflowService().ListTaskQueuePartitions(cctx, request)
+	if err != nil {
+		return fmt.Errorf("unable to list task queues: %w", err)
+	}
+
+	if cctx.JSONOutput {
+		return cctx.Printer.PrintStructured(resp, printer.StructuredOptions{})
+	}
+
+	var items []*taskqueue.TaskQueuePartitionMetadata
+	cctx.Printer.Println(color.MagentaString("Workflow Task Queue Partitions\n"))
+	for _, e := range resp.WorkflowTaskQueuePartitions {
+		items = append(items, e)
+	}
+	_ = cctx.Printer.PrintStructured(items, printer.StructuredOptions{Table: &printer.TableOptions{}})
+
+	items = nil
```

Comment:
```suggestion
	items = items[:0]
```

Don't need to change this, very pedantic, but just thought I'd pass along a nice little Go trick for slice reuse w/ mem savings if you find yourself in other more performance-minded code

Reply:
Nice! I was not aware of that..